tap_dixa/client.py

```diff
@@ -61,7 +62,7 @@ def _post(self, url, headers=None, params=None, data=None):
         return self._make_request(url, method="POST", headers=headers, params=params, data=data)
 
     # Added retry logic for 3 times when bad request or server error or rate limit happens
-    @backoff.on_exception(retry_after_wait_gen, (DixaClient429Error, DixaClient5xxError,DixaClient408Error), jitter=None, max_tries=3)
+    @backoff.on_exception(retry_after_wait_gen, (DixaClient429Error, DixaClient5xxError,DixaClient408Error, ChunkedEncodingError), jitter=None, max_tries=3)
```

Comment:
Inconsistent spacing between exceptions. Since  anyways we are making changes to this line we can refactor it to make it more readable.

```suggestion
    @backoff.on_exception(retry_after_wait_gen,
                                             (DixaClient429Error,  DixaClient5xxError, DixaClient408Error, ChunkedEncodingError),
                                             jitter=None,
                                             max_tries=3)
```